src/components/molecules/Visualizer/Engine/Cesium/useEngineRef.ts

```diff
@@ -84,6 +84,23 @@ export default function useEngineRef(
         if (!viewer || viewer.isDestroyed()) return;
         viewer?.scene?.camera.zoomOut(amount);
       },
+      morphSceneModeTo: (sceneMode, duration = 2) => {
+        const viewer = cesium.current?.cesiumElement;
+        if (!viewer || viewer.isDestroyed() || !viewer.scene) return;
+        switch (sceneMode) {
+          case "scene3d":
+            viewer?.scene?.morphTo3D(duration);
+            break;
+          case "scene2d":
+            viewer?.scene?.morphTo2D(duration);
+            break;
+          case "columbus":
+            viewer?.scene?.morphToColumbusView(duration);
+            break;
+          default:
+            break;
+        }
+      },
```

Comment:
Since `scene3d` is the default, can we write it like this:
```suggestion
        switch (sceneMode) {
          case "scene2d":
            viewer?.scene?.morphTo2D(duration);
            break;
          case "columbus":
            viewer?.scene?.morphToColumbusView(duration);
            break;
          case "scene3d":
          default:
            viewer?.scene?.morphTo3D(duration);
            break;
        }
      },
```
Unless there is a reason you want the default to not do any morphing??